tests/unit/Owncloud/OcisPhpSdk/UserTest.php

```diff
@@ -4,7 +4,7 @@
 
 use OpenAPI\Client\Model\User;
 use Owncloud\OcisPhpSdk\Exception\InvalidResponseException;
-use Owncloud\OcisPhpSdk\User as SdkUser;
+use Owncloud\OcisPhpSdk\BaseUser as SdkUser;
```

Comment:
Why do we need to change this?
`class User extends BaseUser`

So I think that the tests will pass without this change?